lang/cs/Org.Apache.REEF.IMRU/OnREEF/IMRUTasks/MapTaskHost.cs

```diff
@@ -95,10 +98,35 @@ public byte[] Call(byte[] memento)
         }
 
         /// <summary>
+        /// Received close event. 
+        /// If it is closed event sent from driver, set _break to true so that to stop the iteration in Call method
+        /// </summary>
+        /// <param name="value"></param>
+        public void OnNext(ICloseEvent value)
+        {
+            Logger.Log(Level.Info, "The task received close event with message:", value.Value.Value);
```

Comment:
Missing format {0}. Also need to invoke with string, otherwise it will just print the address of the byte array.
 Add a line note